### PR TITLE
[EDU-4521] refactor: add tabs to Marketplace Integrations guides

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-4521] 

### Changes

add tabs to Marketplace Integrations guides.

### Additional links

n/a.

[EDU-4521]: https://aziontech.atlassian.net/browse/EDU-4521?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx [hunk 1]
```diff
@@ -44,6 +70,22 @@ Go to the [building section](/en/documentation/products/guides/build/build-an-ap
...
+
+<Fragment slot="panel.console">
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **Build** section.
+2. On the listing page of your edge applications, select the one you want to use with the **Signed Cookies** integration.
+3. On the **Main Settings** tab, under **Modules**, locate the **Edge Functions** switch and turn it on to enable **Functions** on your edge application.
+4. Still under the **Modules** section, locate the **Application Accelerator** switch and turn it on to enable the **Forward Cookies** functionality for your application.
+  - This will be addressed later at the [Rules Engine section](#configuring-a-rule-on-rules-engine).
+4. Click the **Save** button.
+
+You'll receive a successful message indicating your edge application was updated.
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx:85]: You'll receive a success message indicating that your Edge application has been updated.

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx:85]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx [hunk 2]
```diff
@@ -52,6 +94,9 @@ To configure your integration, proceed as follows:
 4. Click the **Save** button.
 
 You'll receive a successful message indicating your edge application was updated.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/signed-cookies.mdx:96]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx [hunk 3]
```diff
@@ -48,13 +70,31 @@ You can search any integration by browsing through the cards or typing a keyword
...
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
 1. On the upper-left corner, open the **Products menu** > select **Edge Applications** on the **BUILD** section.
 2. On the listing page of your edge applications, select the one you want to use with the Fauna integration.
 3. On the application page, open the **Main Settings** tab.
 4. Enable the **Edge Functions** switch to activate this module on your edge application.
 5. Click the **Save** button.
 
 You'll receive a successful message indicating you've updated your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx:84]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx [hunk 4]
```diff
@@ -48,13 +70,31 @@ You can search any integration by browsing through the cards or typing a keyword
...
+</Fragment>
+
+<Fragment slot="panel.rtm">
+1. On the upper-left corner, open the **Products menu** > select **Edge Applications** on the **BUILD** section.
+2. On the listing page of your edge applications, select the one you want to use with the Fauna integration.
+3. On the application page, open the **Main Settings** tab.
+4. Enable the **Edge Functions** switch to activate this module on your edge application.
+5. Click the **Save** button.
+
+You'll receive a successful message indicating you've updated your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx:94]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx [hunk 5]
```diff
@@ -66,6 +106,26 @@ If a product or module is activated, it could generate usage-related costs. Chec
...
+3. Choose an easy to remember name for your function.
+4. On the dropdown function menu, select the **Fauna** function.
+
+This action will load the **Arguments** tab, where you can add the parameters to execute your application.
+
+This function works without configuration on the **Args** tab. The JSON Args for this function are parametrizable through the request headers: `x-secret` (the key you got on Fauna dashboard) and `x-db-name` (name of the database you're going to create on Fauna).
+
+5. When you're ready, click the **Save** button.
+
+You'll receive a successful message indicating your function was saved.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx:125]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx [hunk 6]
```diff
@@ -87,12 +150,28 @@ Now you have to configure the rules you want (*criteria* and *behavior*) to appl
...
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
 1. Select the **Rules Engine** tab.
 2. Click on **Default Rule**.
     - You can't change the *criteria* field for the **Default Rule** on **Rules Engine**.
 3. On the *behavior* field, select **Run Function** from the dropdown menu and then select the Fauna function, according to the name you gave it in the instantiation step.
 4. Click the **Save** button.
 
 You'll receive a successful message indicating you've updated your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/fauna-marketplace.mdx:164]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/how-to-massive-redirection.mdx [hunk 7]
```diff
@@ -42,21 +64,87 @@ Once you've gotten your integration on **Marketplace**, it'll be available at yo
...
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
 1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
 2. On the listing page of your edge applications, select the one you want to use with the integration.
 3. On the application page, on the **Main Settings** tab, locate the **Edge Functions** switch and turn it on.
 4. Click the **Save** button.
 
 You'll receive a successful message indicating you have updated your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/how-to-massive-redirection.mdx:81]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/how-to-massive-redirection.mdx [hunk 8]
```diff
@@ -42,21 +64,87 @@ Once you've gotten your integration on **Marketplace**, it'll be available at yo
...
+</Fragment>
 
-## Instantiating the integration
+<Fragment slot="panel.rtm">
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
+2. On the listing page of your edge applications, select the one you want to use with the integration.
+3. On the application page, on the **Main Settings** tab, locate the **Edge Functions** switch and turn it on.
+4. Click the **Save** button.
+
+You'll receive a successful message indicating you have updated your edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/how-to-massive-redirection.mdx:90]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/marketplace-content-targeting.mdx [hunk 9]
```diff
@@ -40,12 +62,29 @@ Go to the [building section](/en/documentation/products/guides/build/build-an-ap
...
+    <Fragment slot="tab.console">Console</Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>
+
+<Fragment slot="panel.console">
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
+2. On the listing page of your edge applications, select the one you want to use with the **Content Targeting** integration.
+3. In the **Main Settings** tab, click the **Edge Functions** switch to enable edge functions on your edge application.
+4. Click the **Save** button.
+
+You'll receive a successful message indicating your edge application was updated.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/marketplace-content-targeting.mdx:75]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/marketplace-content-targeting.mdx [hunk 10]
```diff
@@ -40,12 +62,29 @@ Go to the [building section](/en/documentation/products/guides/build/build-an-ap
...
+</Fragment>
+
+<Fragment slot="panel.rtm">
 1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
 2. On the listing page of your edge applications, select the one you want to use with the **Content Targeting** integration.
-3. On the application page, in the **Main Settings** tab, locate and click the **Edge Functions** switch to enable edge functions on your edge application.
+3. In the **Main Settings** tab, click the **Edge Functions** switch to enable edge functions on your edge application.
 4. Click the **Save** button.
 
 You'll receive a successful message indicating your edge application was updated.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/marketplace-content-targeting.mdx:84]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/request-variation.mdx [hunk 11]
```diff
@@ -48,18 +73,99 @@ To create an edge application, go to the [building](/en/documentation/products/g
...
+
+<Fragment slot="panel.console">
+1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
+2. On the listing page of your edge applications, select the one you've created to use with the **Request Variation Controller** integration.
+3. On the **Main Settings** tab, locate the **Edge Functions** switch and turn it on to enable Functions on your edge application.
+4. Then, turn the **Application Accelerator** switch on to enable the *Forward Cookies* functionality for your application.
+    - This will be addressed at the [Rules Engine section](#configuring-a-rule-on-rules-engine).
+5. Click the **Save** button.
+
+You'll receive a message indicating your edge application was successfully updated.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/request-variation.mdx:88]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/request-variation.mdx [hunk 12]
```diff
@@ -48,18 +73,99 @@ To create an edge application, go to the [building](/en/documentation/products/g
...
-2. On the listing page of your edge applications, select the one you've created to use with the Request Variation Controller integration.
-3. On the application page, on the **Main Settings** tab, locate the **Functions** switch and turn it on to enable Functions on your edge application.
-4. Still on the application page, on the **Main Settings** tab, locate the **Application Accelerator** switch and turn it on to enable the *Forward Cookies* functionality for your application; this will be addressed at the [Rules Engine section](#rules-engine).
+2. On the listing page of your edge applications, select the one you've created to use with the **Request Variation Controller** integration.
+3. On the **Main Settings** tab, locate the **Edge Functions** switch and turn it on to enable Functions on your edge application.
+4. Then, turn the **Application Accelerator** switch on to enable the *Forward Cookies* functionality for your application.
+    - This will be addressed at the [Rules Engine section](#configuring-a-rule-on-rules-engine).
 5. Click the **Save** button.
 
 You'll receive a message indicating your edge application was successfully updated.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/request-variation.mdx:99]:
```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**